go/deploy/actions/hostsim.go

```diff
@@ -106,65 +106,71 @@ func RunHostAgentSims(c *pb.DeploymentConfig, remoteTopdir string, showOut bool)
 		c := &simConfigs[i]
 
 		fakeFGs := make([]*pb.FakeFG, len(c.C.FakeFgs))
+		hostsPerNode := (int(c.C.GetNumHostsPerFg()) + len(c.Nodes) - 1) / len(c.Nodes)
 		for j := range fakeFGs {
 			fakeFGs[j] = &pb.FakeFG{
 				SrcDc:            &c.SrcDC,
 				DstDc:            c.C.FakeFgs[j].DstDc,
 				Job:              c.C.FakeFgs[j].Job,
-				MinHostUsage:     proto.Int64(c.C.FakeFgs[j].GetMinFgUsage() / int64(len(c.Nodes))),
-				MaxHostUsage:     proto.Int64(c.C.FakeFgs[j].GetMaxFgUsage() / int64(len(c.Nodes))),
+				MinHostUsage:     proto.Int64(c.C.FakeFgs[j].GetMinFgUsage() / int64(c.C.GetNumHostsPerFg())),
+				MaxHostUsage:     proto.Int64(c.C.FakeFgs[j].GetMaxFgUsage() / int64(c.C.GetNumHostsPerFg())),
 				ApprovalBps:      proto.Int64(c.C.FakeFgs[j].GetApprovalBps()),
 				TargetNumSamples: proto.Int32(c.C.FakeFgs[j].GetTargetNumSamples()),
 			}
 		}
 
 		const startHostID = 100001
 		numHosts := int32(0)
-		hostsPerNode := (int(c.C.GetNumHostsPerFg()) + len(c.Nodes) - 1) / len(c.Nodes)
+		nProcesses := 10
+		
 		for nidx, n := range c.Nodes {
 			n := n
+			hostsPerProcess := hostsPerNode/nProcesses
```

Comment:
Not sure this works if hostsPerNode doesn't divide evenly into nProcesses, but also I don't care too much. Maybe just add a check that there's 0 remainder and issue an error if so?

Compare with hostsPerNode that rounds up.